.github/styles/config/vocabularies/Polkadot/accept.txt

```diff
@@ -0,0 +1,26 @@
+args
+backchannel
+blockHash
+bootnode
+bootnodes
+config
+[Cc]oretime
+dev_newBlock
+dev_setHead
+dev_setStorage
+dev_timeTravel
```

Comment:
were you able to figure out a way for ignoring the stuff in quotes, like `??? function "THIS STUFF"`

Reply:
Not yet. I got it to ignore the params in those instances but still working on the function names. Haven't landed on the regex pattern yet that won't also grab other stuff.

Reply:
Once I have the vocab beefed up (need to screen more files) and get the existing rules in place, I will start making a list of things from the style guide still needing enforced and open some tickets so we can track them as they are added to the config, etc.